portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -36,41 +38,38 @@
 
   private final Gson gson = new Gson();
 
-  /**
-   * Gets the comments from datastore
-   * @param request - client asks information from servlet
-   * @param response - servlet writes response
-   * @return send the response back to client
-   */
   @Override
   public void doGet(final HttpServletRequest request, final HttpServletResponse response) throws IOException {
 
     Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     PreparedQuery results = datastore.prepare(query);
 
+    // Limit on the number of comments to get from datastore.
+    String userLimit = request.getParameter("limit");
+    FetchOptions limit;
```

Comment:
nit: fetchOptions or fetchOptionsWithLimit seems like a better name for this variable.

Reply:
Done :)